libs/components/colorpicker/src/lib/modules/colorpicker/types/colorpicker-hsva.ts

```diff
@@ -1,27 +1,27 @@
 // spell-checker:ignore hsva,colorpicker
 /**
- * Specifies colors as a combination of hue, saturation, and value with an alpha
+ * Colors as a combination of hue, saturation, and value with an alpha
```

Comment:
```suggestion
 * Colors specified as a combination of hue, saturation, and value with an alpha
```
Same suggestion as before about sticking with `Specifies colors as...` here.